supplementary_style_guide/introduction/whats-new.adoc

```diff
@@ -32,6 +32,107 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
+* *xref:infrastructure-node[infrastructure node]*: Added glossary entry
+* *xref:initiator[initiator]*: Added glossary entry
+* *xref:internal-mode[internal mode]*: Added glossary entry
+* *xref:local-storage[local storage]*: Added glossary entry
+* *xref:logical-cache[logical cache]*: Added glossary entry
+* *xref:logical-volume[logical volume]*: Added glossary entry
+* *xref:metadata[metadata]*: Added glossary entry
+* *xref:multipath[multipath]*: Added glossary entry
+* *xref:network-encryption[network encryption]*: Added glossary entry
+* *xref:network-time-configuration[Network Time Configuration]*: Added glossary entry
+* *xref:n-way-replication[n-way replication]*: Added glossary entry
+* *xref:object-storage[object storage]*: Added glossary entry
+* *xref:ocp-cluster[OpenShift Container Platform cluster]*: Added glossary entry
+* *xref:phase[phase]*: Added glossary entry
+* *xref:physical-disk[physical disk]*: Added glossary entry
+* *xref:physical-volume[physical volume]*: Added glossary entry
+* *xref:platform[platform]*: Added glossary entry
+* *xref:primary-cluster[primary cluster]*: Added glossary entry
+* *xref:primary-node[primary node]*: Added glossary entry
+* *xref:provision[provision]*: Added glossary entry
+* *xref:proxy[proxy]*: Added glossary entry
+* *xref:replicated-volume[replicated volume]*: Added glossary entry
+* *xref:restore[restore]*: Added glossary entry
+* *xref:pm-ostree[pm-ostree]*: Updated glossary entry to add formatting guidance
+* *xref:scale-out[scale out]*: Added glossary entry
+* *xref:scale-up[scale up]*: Added glossary entry
+* *xref:secondary-cluster[secondary cluster]*: Added glossary entry
+* *xref:self-heal[self-healing]*: Added glossary entry
+* *xref:server[server]*: Added glossary entry
+* *xref:snapshot[snapshot]*: Added glossary entry
+* *xref:source-node[source node]*: Added glossary entry
+* *xref:source-volume[source volume]*: Added glossary entry
+* *xref:split-brain[split brain]*: Added glossary entry
+* *xref:storage-cluster[storage cluster]*: Added glossary entry
+* *xref:storage-pool[storage pool]*: Added glossary entry
+* *xref:stripe[stripe]*: Added glossary entry
+* *xref:subvolume[subvolume]*: Added glossary entry
+* *xref:target[target]*: Updated glossary entry to include a generic definition
+* *xref:target-volume[target volume]*: Added glossary entry
+* *xref:thickly-provisioned[thickly provisioned]*: Added glossary entry
+* *xref:total-capacity[total capacity]*: Added glossary entry
+* *xref:translator[translator]*: Added glossary entry
+* *xref:usable-capacity[usable capacity]*: Added glossary entry
+* *xref:virtual-disk[virtual disk]*: Added glossary entry
+* *xref:volume-file[volume file]*: Added glossary entry
+* *xref:volume-group[volume group]*: Added glossary entry
+* *xref:worker-node[worker node]*: Added glossary entry
+
+.Style guidance
+* *xref:commands-with-root-privileges[Commands requiring root privileges]*: Added guidance to include an example
```

Comment:
Should this be "updated" instead of "added", since it's not a new entry?

```suggestion
* *xref:commands-with-root-privileges[Commands requiring root privileges]*: Updated guidance to include an example
```